fe/java-udf/src/main/java/org/apache/doris/udf/JdbcExecutor.java

```diff
@@ -248,9 +248,11 @@ private void init(String driverUrl, String sql, int batchSize, String driverClas
 
             dataSource = new HikariDataSource(config);
             conn = dataSource.getConnection();
+            conn.setAutoCommit(false);
             if (op == TJdbcOperation.READ) {
```

Comment:
should set it to false only the op==READ

Reply:
done